vanilla/src/mixins/java/org/spongepowered/vanilla/mixin/core/entity/player/ServerPlayerEntityMixin_Vanilla.java

```diff
@@ -81,6 +83,18 @@
         }
     }
 
+    /**
+     * @author dualspiral - 18th December 2020 - 1.16.4
+     * @reason This is effectively an overwrite, redirecting the vanilla
```

Comment:
If it is effectively an overwrite, make it one. With auditing, these overwrites like up like a Christmas tree, ensures we don't overlook it.